app/controllers/papers_controller.rb

```diff
@@ -150,4 +161,34 @@ def prevent_update_on_locked!
       raise ActiveRecord::RecordInvalid, paper
     end
   end
+
+  def notify_paper_created!
+    ActivityFeed.create(
+      feed_name: 'manuscript',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.